### ISSUE TITLE
getTracks doesn't work anymore

### ISSUE THREAD

**Issue by robotzurg** [microlinkhq/spotify-url-info#113]
**Describe the bug**
getTracks() gives an error when being run on this album

**Spotify URL(s)**
https://open.spotify.com/album/6YuNt7ueupL1klxLJiTqTJ?si=HLEtPLJoTaqprBmRm7qkCQ

**Your Code**
(trackLink is a variable holding that link above)
```
if (trackLink.includes('album')) {
    const tracks = await getTracks(trackLink);
    console.log(tracks);
}
```

**Error Message**
TypeError: Cannot read properties of undefined (reading 'items')
    at getTracks (/Users/jeffturpin/Documents/GitHub/Waveform/discord-bot/node_modules/spotify-url-info/src/index.js:145:26)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async Object.execute (/Users/jeffturpin/Documents/GitHub/Waveform/discord-bot/commands/sendmail.js:71:28)
    at async Client.<anonymous> (/Users/jeffturpin/Documents/GitHub/Waveform/discord-bot/index.js:225:9)

**Comment by MementoAequitas** [comment]
getting similar errors. issue with recent update?
```
TypeError: Couldn't find any data in embed page that we know how to parse.
Please report the problem at https://github.com/microlinkhq/spotify-url-info/issues.
at throwError (/opt/node_modules/spotify-url-info/src/index.js:22:9)
at /opt/node_modules/spotify-url-info/src/index.js:65:10
at processTicksAndRejections (internal/process/task_queues.js:95:5)
at async Function.addPlaylist (/var/task/helper/player.js:97:14)
at async eval (/functions/events/discord/message/create.js:59:3)


TypeError: Cannot read property 'items' of undefined
at getTracks (/opt/node_modules/spotify-url-info/src/index.js:145:26)
at processTicksAndRejections (internal/process/task_queues.js:95:5)
at async Function.addPlaylist (/var/task/helper/player.js:97:14)
at async eval (/functions/events/discord/message/create.js:59:3)
```

**Comment by CaptSpider4** [comment]
![image](https://user-images.githubusercontent.com/59257790/208959331-48da4fe7-1df4-469a-9c63-85688bc29937.png)

Facing similar issues v3.1.9

**Comment by robotzurg** [comment]
Yeah, sounds like a recent spotify update broke it :/

**Comment by MementoAequitas** [comment]
I use Spotify and exchange over to yt. however Im using getData(song) from Spotify.
 
 Received a similar notification that @CaptSpider4 posted.
 
  ```
  static async play(song, {channel_id, guild_id, direct, keyDetails} = {}) {
    if (song?.match(/^https?:\/\/(open.spotify.com|spotify.com)(.*)$/)) {
      if(song.includes('playlist') || song.includes('album')) return this.addPlaylist(song, {channel_id, guild_id, keyDetails});
      else if(song.includes('track')) song = await getData(song)
```

**Comment by Danrejk** [comment]
Same thing for me.
`TypeError: Cannot read properties of undefined (reading 'items')`
Tried fixing it, but this bug report does explain a lot. The getTracks just simply don't work. That's why I was getting undefined from "tracks"

**Comment by Kikobeats** [comment]
Hello,

I pre-released [spotify-url-info@3.2.0-0](https://github.com/microlinkhq/spotify-url-info/releases/tag/v3.2.0-0) and it should to address the issue.

Can you confirm it's working fine to you?

**Comment by MementoAequitas** [comment]
getting this new error since update:
```
TypeError: Cannot read property '0' of undefined
at /var/task/helper/player.js:102:50
at Array.map (<anonymous>)
at /var/task/helper/player.js:99:17
at processTicksAndRejections (internal/process/task_queues.js:95:5)
at async Function.addPlaylist (/var/task/helper/player.js:97:14)
at async eval (/functions/events/discord/channel/message/create.js:24:3)
```

which is associated with this little bit of code:
```js
    if (url.includes('spotify')) {
      list = await getTracks(url).then(
        (data) =>
          data?.map((x) => {
            return {
              title: x.name,
              shortUrl: `${x.name} by ${x.artists[0].name} audio`,
            };
          }) || []
      );
    } else {
      list = await ytpl(url, {limit: process.env.PLAYLIST_LIMIT || 20}).then(
        (res) => {
          return res.items;
        }
      );
    }
 ------------------------------------------------------------------   
>>>    shortUrl: `${x.name} by ${x.artists[0].name} audio`,
```

**Comment by Kikobeats** [comment]
please provide a way to reproduce it; what's the URL?

**Comment by robotzurg** [comment]
> Hello,
> 
> I pre-released [spotify-url-info@3.2.0-0](https://github.com/microlinkhq/spotify-url-info/releases/tag/v3.2.0-0) and it should to address the issue.
> 
> Can you confirm it's working fine to you?

After adjusting how my code works, this does work, thank you!
